antora-playbook.preview.local.yml

```diff
@@ -0,0 +1,9 @@
+site:
+  title: Couchbase Docs Local Preview
+output:
+  dir: ./preview
+  clean: true
+asciidoc:
+  attributes:
+    kroki-server-url: null
```

Comment:
Not sure what this does, to be honest, but doesn't it need a setting?

Reply:
`null` gets set to `~` in YAML, which I *think* is the same as being unset.
(At any rate, the builds I tried didn't hang while trying to speak to our firewalled server, so I think it's OK)